lib/ThinEvent.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Stripe;
+
+/**
+ * @property string             $id Unique identifier for the event.
+ * @property string             $type
+ * @property string             $created
+ * @property null|string        $context
+ * @property null|RelatedObject $related_object
+ * @property null|Reason        $reason
```

Comment:
We can add better doc strings here. Refer to https://github.com/stripe/stripe-node/blob/934e0fa98ccdfa7b5fb715f4bc4e9f8e3b76bb4e/types/V2/Events.d.ts#L37